region/brinstar/blue.json

```diff
@@ -135,38 +131,84 @@
           "from": 1,
           "to": [
             {
-              "id": 5,
+              "id": 6,
               "requires": [
-                "Morph",
-                {"obstacle":{
-                  "id": "A",
-                  "requires": [
-                    {"or":[
-                      "Bombs",
-                      {"and":[
-                        "PowerBomb",
+                {"or":[
+                  {"cost":[
+                    {"cost":[
+                      {"enemyDamage": {
+                        "enemy": "Sidehopper",
+                        "type": "contact",
+                        "hits": 1
+                      }}
+                    ]}
+                  ]},
+                  {"obstacle":{
+                    "id": "C",
+                    "requires": [
+                      {"or":[
                         {"cost":[
-                          {"ammo": [
-                            {
-                              "type": "PowerBomb",
-                              "count": 1
-                            }
+                          {"enemyDamage": {
+                            "enemy": "Sidehopper",
+                            "type": "contact",
+                            "hits": 8
```

Comment:
This one is tough to balance. I'm unsure about it but 8 does seem a bit excessive. https://youtu.be/7l-mDFOryYc

Reply:
Yeah it's intended as a "not-quite-worst-case-but-things-didn't-go-well" scenario.

Reply:
I don't know to what extent it matters, though. This can literally never be logically required.

Reply:
How about 6 instead?

Reply:
Sure